internal/test/scorecard/common_utils.go

```diff
@@ -485,18 +500,25 @@ func updateAndWaitTillCryostatAvailable(cr *operatorv1beta1.Cryostat, resources
 	return err
 }
 
-func cleanupCryostat(r *scapiv1alpha3.TestResult, client *CryostatClientset, name string, namespace string) {
+func cleanupAndLogs(r *scapiv1alpha3.TestResult, client *CryostatClientset, name string, namespace string, ch *chan *ContainerLog) {
```

Comment:
I think here we can use `chan *ContainerLog`. The channel itself is already a reference so we can pass `nil` if not available.

Reply:
Maybe name the parameter a bit more descriptive? Something like: `logChannel`?